src/ConfigEditor/AuthTypeSwitch.tsx

```diff
@@ -23,8 +23,7 @@ export function AuthTypeSwitch({ useManagedSecret, onChangeAuthType }: Props) {
           </div>
         ) : (
           <div style={{ marginTop: '10px', marginBottom: '10px', minWidth: '670px' }}>
-            Use the <code>GetClusterCredentials</code> IAM permission and your database user to generate temporary
```

Comment:
When using temporary credential authentication provisioned Redshift will use  [GetClusterCredentials](https://docs.aws.amazon.com/redshift/latest/APIReference/API_GetClusterCredentials.html) wheras Redshift serverless will use [GetCredentials](https://docs.aws.amazon.com/redshift-serverless/latest/APIReference/API_GetCredentials.html) API. 

I had a choice to show this difference in the message but chose to simplify. 
It's because this is a minor difference in the background which users do not need take care about.

Reply:
I see. I updated the description at 2faaa0fce7584070f80632e8b43bf1d97457cf3c.